app/controllers/planning_applications/review/conditions_controller.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Review
+    class ConditionsController < AuthenticationController
+      include CommitMatchable
+      include PlanningApplicationAssessable
+
+      before_action :set_planning_application
+      before_action :ensure_planning_application_is_validated
+      before_action :ensure_user_is_reviewer
+      before_action :set_condition_set
+      before_action :set_condition_set_review
+
+      def show
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def edit
+        respond_to do |format|
+          format.html
+        end
+      end
```

Comment:
The `show` and `editable` views are the same apart from `editable` being passed to the `form` partial being `true/false` - we should only have the `show` action and use the status to determine the value of `editable`.